### PR TITLE
2272 UI Ember user daily statistic table ember

### PR DESCRIPTION
**Pull request by koylubaevnt**
[EPMLABSBRN-XX](https://jira.epam.com/jira/browse/EPMLABSBRN-XX)

**Description**:

### CODE REVIEW

File: frontend/app/components/statistics/bar-chart/component.ts [hunk 1]
```diff
@@ -29,6 +31,8 @@ export default class BarChartComponent extends Component<IBarChartComponentArgs>
     return this.args.data;
   }
 
+  @tracked selectedIndex?: number | null;
```
Comment (lifeart) [inline frontend/app/components/statistics/bar-chart/component.ts:34]: looks like this prop is not used

**Review by lifeart — APPROVED** [review]

**Comment by koylubaevnt** [comment]
@lifeart recheck please PR. I made some fixes...

File: frontend/app/components/statistics/bar-chart/component.ts (hunk 1 above)
Comment (koylubaevnt) [reply frontend/app/components/statistics/bar-chart/component.ts]: Yes. I removed it.

**Review by lifeart — CHANGES_REQUESTED** [review]
removing approval as requested

**Review by lifeart — APPROVED** [review]

**Pull request merged**